src/DynamicForm.vue

```diff
@@ -70,7 +47,7 @@
             default: true
```

Comment:
```suggestion
            default: ""
```
Do you mean empty string here or something like that? Because otherwise it'll be the wrong type no?

Reply:
I have not changed the behaviour and I don't intend to do so. This is just purely an upgrade, will deal with improvement later once everything works.

Reply:
Not sure if github is displaying this weirdly, but this looks like a comment on `includeSubmitButton` which is a Boolean so `true` seems ok to me anyway..?